src/configurations/destinations/gainsight_px_browser/db-config.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "GAINSIGHT_PX_BROWSER",
+  "displayName": "Gainsight PX",
+  "config": {
+    "transformAtV1": "processor",
+    "saveDestinationResponse": true,
+    "includeKeys": [
+      "apiKey",
+      "dataCenter",
+      "extraConfig",
+      "blacklistedEvents",
+      "whitelistedEvents",
+      "oneTrustCookieCategories",
+      "eventFilteringOption",
+      "consentManagement"
+    ],
+    "excludeKeys": [],
+    "supportedSourceTypes": ["web"],
+    "supportedConnectionModes": { "web": ["device"] },
+    "supportedMessageTypes": { "device": { "web": ["identify", "track", "group"] } },
+    "destConfig": {
+      "defaultConfig": [
+        "apiKey",
```

Comment:
`apiKey` is missing in ui-config.json

Reply:
Fixed, thanks, productKey is the correct name, I fixed it.